LabeledItem/LabeledItem.js

```diff
@@ -27,7 +27,7 @@ import Item from '../Item';
  */
 const LabeledItemBase = deprecate(
 	({children, titleIcon, ...rest}) => (
-		<Item {...rest}>
+		<Item role="button" {...rest}>
```

Comment:
I defined the requirement to read `button` before in a LabeleddItem. But I think that we might not do it. Please remove this modification in this line. Sorry for the inconvenience.